src/components/TodosList.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable max-len */
 /* eslint-disable react/prop-types */
```

Comment:
eslint assist with catching errors in our code base, kindly avoid disabling eslint